diagnostic.md

```diff
@@ -20,14 +20,25 @@ machine serving the donuts is `http://www.example.com`.
 Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
-```sh
-# your answer here
+```curl --include --request GET "http://example.com/donuts"
+
 ```
 
 Write an AJAX request to retrieve a list of **all donuts** on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = $('#donut-id').val();
+
+donutApi.index()
+  .then(donutUi.logResponseBody)
+  .catch(donutUi.logRequestError);
+
+const index = function () {
+  return $.ajax({
+    url: app.host + '/donuts',
+    method: 'GET',
+  });
+};
 ```
```

Comment:
Suja, you need to think simpler!

This is advanced and shows a strong understanding of the content, but it assumes things like index and app.host are defined, within the context of this question they aren't.

```js
let getDonuts = function () {
  return $.ajax({
    url: 'http://www.example.com/donuts',
    method: 'GET',
  });
};
``` 

^this is all we needed here